app/models/problem_report.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: problem_reports
+#
+#  id                  :integer          not null, primary key
+#  reporter_user_id    :integer
+#  last_update_user_id :integer
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  venue               :string
```

Comment:
Might want to change this to place, and then adding `has_one :place` for consistency